src/common/components/repository-row/index.js

```diff
@@ -376,11 +391,9 @@ RepositoryRow.propTypes = {
   registerNameStatus: PropTypes.shape({
     success: PropTypes.bool
   }),
-  dispatch: PropTypes.func.isRequired,
-  registerNameIsOpen: PropTypes.bool
+  dispatch: PropTypes.func,
+  registerNameIsOpen: PropTypes.bool,
+  configureIsOpen: PropTypes.bool
 };
 
-// FIXME:
-// `connect` shouldn't be used just to add dispatch to props
-// this makes it much harder to test it as it get wrapped
-export default connect()(RepositoryRow);
+export default connect()(RepositoryRowView);
```

Comment:
What I don't like about this approach is that using empty `connect` call doesn't explicitly define what is connected (what part of state is passed to the component).

I know it's just to pass `dispatch`, but I would prefer it to be more explicit in `connect` call, for example by using `mapDispatchToProps`.